### ISSUE TITLE
🐛 Overlapping parts

### ISSUE THREAD

**Issue by Dexus** [deepnest-next/deepnest#11]
https://github.com/deepnest-io/Deepnest/issues/145

![Screenshot_2024-12-23_09-58-51](https://github.com/user-attachments/assets/d5719618-c694-4578-b7be-ad07af44102a)
[bug-test-overlap.zip](https://github.com/user-attachments/files/18228099/bug-test-overlap.zip)

**Comment by Dexus** [comment]
@ClockworkBastard is it possible that this only happends when the "Use rough approximation" is enabled?

I did run 30.000.000 tests now and I was now able to get one result that would overlap with the option enabled.

**Comment by KoreUrala** [comment]
Hi! I just found this project out of having this exact issue!
Deepnest is a life saver for my sewing projects because it allows me to accommodate a myriad of different patterns in a single sheet, so first of all, thanks for your efforts!
Secondly, i'm having overlapping parts without the rough approximation feature :(

![Image](https://github.com/user-attachments/assets/8bd96888-6ba2-4483-a82c-a756f9483106)

this happened while trying to fit five of these bad boys

![Image](https://github.com/user-attachments/assets/7b2efb7e-257f-4f51-a831-0a6d65538394)

in laser cut table (130cm * 90cm) dimensions.
I was using everything on default save for:
Part rotations - 2
Optimization type - Bounding box
Optimization ratio - 0.01
GA population - 256
GA mutation rate - 16

Using Windows 11 Pro running in an AMD Ryzen7 2700X along with a NVIDIA GeForce RTX 3060 with 16GB RAM
Dunno if the pc specs are relevant though...

**Comment by Dexus** [comment]
@KoreUrala Thank you. Can you share the settings completely?  Is the SVG Preprocessor acativated?

**Comment by KoreUrala** [comment]
SVG Prewhat...? Uhhh... idunno, it's probably on default...

Here, have a ss of the settings
![Image](https://github.com/user-attachments/assets/c98e49de-cc13-4b16-aa24-cfe13e254213)
Also this
![Image](https://github.com/user-attachments/assets/48fa1537-7731-475c-ac99-8927d9ca1924)
Lemme run it again...
-_time skip_-

![Image](https://github.com/user-attachments/assets/a1895c4c-730c-4b8e-8d69-bc5827026071)
It infuriates me to no end...

**Comment by KoreUrala** [comment]
Yeah, still happens with normalizer, I also noticed it behaves slightly better in inches than in millimeters... very slightly...

**Comment by Dexus** [comment]
Thank you for letting me know that. Hope that I can find the "bad" part now,

**Comment by Dexus** [comment]
Thank you @KoreUrala! I have now been successful in recreating the behaviour and am currently finalising the fix. Will be released with v1.4.1. I'm running some tests now, and if there are no more overlaps, it will be released tonight or tomorrow during the day.
